src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -51,7 +51,15 @@ object DepositProperties {
     val s = readProperties(f)
     log.debug(s"[$id] Trying to retrieve state from $f")
     if(!f.exists()) throw new IOException(s"$f does not exist")
-    State(s.getString("state.label"), s.getString("state.description"), new DateTime(s.getFile.lastModified()).withZone(DateTimeZone.UTC).toString)
+    val state = Option(s.getString("state.label")).getOrElse("")
```

Comment:
You could also just use [containsKey](http://docs.oracle.com/javase/8/docs/api/java/util/Hashtable.html#containsKey-java.lang.Object-)